README.md

```diff
@@ -144,6 +150,10 @@ funcsave grlz
 funcsave gdoc
 funcsave gtst
 ```
+Alternative function for Linux users for IMPROVE in fish shell:
```

Comment:
Same goes here, add one extra line with a small comment inside the code snippets.